src/Kiota.Builder/Plugins/PluginsGenerationService.cs

```diff
@@ -752,4 +760,44 @@ rExtRaw is T rExt &&
 
         return null;
     }
+
+    private static ResponseSemantics? GetResponseSemanticsFromTemplate(OpenApiOperation openApiOperation)
+    {
+
+        if (openApiOperation.Responses is null || openApiOperation.Responses.Count == 0)
+        {
+            return null;
+        }
+
+        if (!openApiOperation.Responses.TryGetValue("200", out var response) || response is null)
+        {
+            return null;
+        }
+
+        if (response.Content is null || response.Content.Count == 0)
+        {
+            return null;
+        }
+
+        var schema = response.Content["application/json"]?.Schema;
+        if (schema is null)
+        {
+            return null;
+        }
+
+        var staticTemplateNode = new JsonObject
+        {
+            ["file"] = "./adaptive-card.json"
+        };
```

Comment:
why are we using the object model at all here? it could simply be a resource string to avoid the serialization step.